3_harmonize/src/summarize_targets.R

```diff
@@ -0,0 +1,39 @@
+#' @title Summarize pipeline metadata
+#' 
+#' @description 
+#' Function to summarize and save pipeline metadata
+#'
+#' @param names character string indicating which targets to return pipeline
+#' metadata for. Defaults to NULL, which returns all targets.
+#' @param fileout character string indicating the name of the saved file, 
+#' including file path and extension.
+#' 
+#' @returns 
+#' Returns a csv file containing pipeline metadata, with a row for each 
+#' requested `tar_name` and columns representing a unique data hash, the
+#' date of last build, and the build time for each target in seconds. 
+#' 
+summarize_targets <- function(names = NULL, fileout) {
```

Comment:
I think it would make more sense to switch the argument order. It can be tricky to have a required argument come _after_ an arg with a default because a user must always use the argument name when calling the function. If someone ran `summarize_targets('file_out.csv')` it would fail. I think it breaks this common usage pattern and switching the args would prevent issues.

Reply:
Yep, good catch! I've switched the argument order in https://github.com/USGS-R/ds-pipelines-targets-example-wqp/pull/80/commits/293857d240ae87ca951eadf33977444d24a1f1ce.